apis/config/v1beta1/configuration_types.go

```diff
@@ -273,8 +273,24 @@ type MultiKueue struct {
 	// Defaults to 15 minutes.
 	// +optional
 	WorkerLostTimeout *metav1.Duration `json:"workerLostTimeout,omitempty"`
+
+	// DispatcherName defines the dispatcher responsible for selecting worker clusters to handle the workload.
+	// - If specified, the workload will be handled by the named dispatcher.
+	// - If not specified, the workload will be handled by the default (MultiKueueDispatcherModeAllAtOnce) dispatcher.
```

Comment:
```suggestion
	// - If not specified, the workload will be handled by the default ("kueue.x-k8s.io/multikueue-dispatcher-all-at-once") dispatcher.
```

API comments should use the user-facing API name.